CONTRIBUTING.md

```diff
@@ -79,6 +79,7 @@ Similarly, please create one PR per development item, instead of bundling multip
 * Flutter comes with **Hot Reload** (the lightning icon, or *Ctrl+\\* or *⌘\\*), which allows you to load changes in the code quickly into an already running app, without you needing to reinstall it. It's a very handy feature, but it doesn't work all the time - if you change the code, use Hot Reload but don't see the expected changes, or see some weird behaviour, you may need to close and restart the app (or even reinstall).
 * If **running on web** doesn't give the expected results after changing some code, you may need to clear the cache (in *Chrome*: *Ctrl+Shift+C* or *⌘+Shift+C* to open the Inspect menu, then right-click the *Refresh* button, and select *Empty cache and Hard reload*.)
 * [Flutter Inspector](https://flutter.dev/docs/development/tools/devtools/inspector) is a powerful tool which allows you to visualize and explore Flutter widget trees. You can use it to find out where a specific part of the UI is defined in the code (by turning on *Select widget mode* and selecting the widget you'd like to find), it can help you debug layouts (by enabling *Debug Paint*, you can visualize padding, alignments and widget borders) and much more.
+  - If *Flutter Inspector* doesn't work in *Web* mode a possible solution is to disable *embdedding DevTools* (IntelliJ/Android Studio: File>Settings>Languages & Frameworks>Flutter>**Enable embedding DevTools in the Flutter Inspector tool window**).
```

Comment:
The formatting is a bit odd imo and you have a typo. Also, "web mode" isn't really a thing we say, since web is not a mode, but rather just a platform we're building the code for. How about this:
```suggestion
  - If *Flutter Inspector* doesn't work when running on web, a possible solution is to disable *embedding DevTools* (IntelliJ/Android Studio: `File > Settings > Languages & Frameworks > Flutter > Enable embedding DevTools in the Flutter Inspector tool window`).
```